packages/server/src/fhir/routes.test.ts

```diff
@@ -441,11 +452,23 @@ describe('FHIR Routes', () => {
     expect(res.status).toBe(200);
   });
 
-  test('Search', async () => {
+  test.each<['writer' | 'reader']>([['writer'], ['reader']])('Search on %s', async (repoMode) => {
```

Comment:
Rather than repeating this `.each()` on every test, I'd recommend wrapping the tests in a `describe.each()`